### DISCUSSION TITLE
MDN Plus - Collections Page

### DISCUSSION THREAD

**Discussion by ghost** [mdn/mdn-community#192] [MDN Plus] [answered]
Hi MDN Team, 

I'm a new member to the MDN plus service. I was wondering whether there was any work in the pipeline to create named collections within the collections area. 

At the moment, I can save a bunch of pages to the 'Collections' page, but it's very quickly becoming quite messy. I think a folder like structure would be great. 

For example, I'd love to save the Javascript Array Methods pages to an Array Methods collection.

Hope this is the right forum to bring this up.

Thanks, Michael

**Comment by Guyzeroth** [comment, marked as answer]
Hi Michael.

Thanks for your input! 

This is currently in development and will be previewed to MDN supporters in the coming weeks :) . It'd be great to get your feedback once it is launched!

Cheers

Guy
